PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,8 @@
+## Description
+<!-- Expectations for PRs: [RFC66 - Pull Request & Code Review Standards](https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards) -->
```

Comment:
Markdown isn't rendered in comments, so it's probably cleaner to use the bare URL.
```suggestion
<!-- Expectations for PRs: https://safetyculture.atlassian.net/wiki/spaces/ENG/pages/2881716914/RFC66+Pull+Request+Code+Review+Standards -->
```